### ISSUE TITLE
CORS blocking automation

### ISSUE THREAD

**Issue by denzuko** [Dallas-Makerspace/calendar#47]
* **Expected Results**:
  - Able to use JavaScript to create an event

* **Actual Results**:
  - CORS blocks requests from development IDE at `s.codepen.io`

* **Additional Details**:
   - Console.log Message: 
      ```Failed to load https://calendar.dallasmakerspace.org/events/add: Redirect from 'https://calendar.dallasmakerspace.org/events/add' to 'https://calendar.dallasmakerspace.org/users/login' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://s.codepen.io' is therefore not allowed access.```
   - OS: Windows server 2012 (jumpbox.local)
   - Browser: Chrome v61.0.3163.100
   - [Example Codepen](https://codepen.io/devopscast/pen/yPVwye)

**Comment by MikeColeGuru** [comment]
We might should develop an API for doing this.  I believe the API is already in CakePHP, it just needs to be configured and turned on.. I think

**Comment by limey-dms** [comment]
Updating/closing the old issues as stale/won't fix
